acquia_lift_profiles/js/acquia_lift_profiles.debug.js

```diff
@@ -0,0 +1,124 @@
+/**
+ * Defines JavaScript functions needed by the Acquia Lift debugger in
+ * order to provide customer profile information.
+ */
+
+(function ($, Drupal) {
+
+  Drupal.acquiaLiftProfilesDebug = (function() {
+    var debugPrefix = "acquiaLift::debug::"
+    var curSegmentCapture = {};
+    var curSegments = [];
+    var curSegmentsOverride;
+    var curIdentities = [];
+    var message;
+    var code;
+    $(document).on("segmentsUpdated", function( event, data, capture ) {
+      curSegmentCapture = capture;
+      if(data){
+        curSegments = data["segments"];
+        curSegmentsOverride =  JSON.parse(window.sessionStorage.getItem(debugPrefix + "overrideSegments"));
+        if ( curSegmentsOverride ) {
+          if(data["segments"]){
+            data["segments"].length = 0;
+            $(curSegmentsOverride).each( function(index,overrideSegment) { data["segments"].push(overrideSegment); } );
+          }
+          curSegments = curSegmentsOverride.splice(0);
+
+        }
+        message = "Segments Returned: " + curSegments;
+        code = 1000;
+      }else{
+        message = "No Data found"
+        code = 3001;
+      }
+      Drupal.personalizeDebug.log( message , code, 'Lift Web');
+    });
+    $(document).bind("identitiesAdded", function( event, identities ) {
+      // TODO: Check if the person id changed because TC_CONF.userIdentitySourceInTrackingId is true
+      if(identities){
+        $.each(identities, function (index, identity) { curIdentities.push( identity ); } );
+      }
+    });
+
+    return {
+      'getPersonId' : function() {
+        return $.cookie("tc_ptid");
+      },
+
+      'getTouchId' : function() {
+        return $.cookie("tc_ttid");
+      },
+
+      'isThirdPartyPersonId' : function() {
+        return TC_CONF && TC_CONF.thirdPartyCookie == true;
```

Comment:
Where is TC_CONF coming from?  Comment explanation is fine and needed mostly because it isn't ever mentioned in the file otherwise.